src/pg4n/config_parser.py

```diff
@@ -0,0 +1,72 @@
+import re
+from typing import Optional, TextIO
+
+from .config_values import ConfigValues
+
+
```

Comment:
Nipotusta: voidaan siirtää tänne `re.compile(...)`-kutsu, tällä hetkellä antaa vaikutelman että regex-lauseke olisi parsintakohtainen.
```suggestion
_option_matcher = re.compile(
    r"[ \t]*(?P<optname>\w+)[ \t]+(?P<optval>1|0|true|false|yes|no)[ \t]*$",
    flags=re.IGNORECASE,
)


```